webrtc.html

```diff
@@ -12001,8 +12001,8 @@ <h3 id="sec-constraints">Constraints</h3>
       default values for attributes that would be left uninitialized without
       the init dictionary present.</li>
       <li>InvalidMediaStreamTrackError changed to InvalidParameter.</li>
-      <li>Fire NetworkError when the data transport is closed with an
-      error.</li>
+      <li>Fire the <code><a>RTCError</a></code> event
+      when the data transport is closed with an error.</li>
```

Comment:
@taylor-b  It's probably better to add a new attribute for the SCTP Cause Code (e.g. sctpCauseCode) rather than using errorDetail. 

For data channel errors (e.g. above the transport layer), I am wondering if it makes sense to use the close event with a reason code, such as is done with WebSockets:
https://html.spec.whatwg.org/multipage/comms.html#handler-websocket-onclose

"When the WebSocket connection is closed, possibly cleanly, the user agent must queue a task to run the following substeps:
Change the readyState attribute's value to CLOSED (3).
If the user agent was required to fail the WebSocket connection, or if the the WebSocket connection was closed after being flagged as full, fire an event named error at the WebSocket object. [WSP]
Fire an event named close at the WebSocket object, using CloseEvent, with the wasClean attribute initialized to true if the connection closed cleanly and false otherwise, the code attribute initialized to the WebSocket connection close code, and the reason attribute initialized to the result of applying UTF-8 decode without BOM to the WebSocket connection close reason. [WSP]"